### PR TITLE
Integrate price feeder

### PR DESCRIPTION
**Pull request by zarazan**
- Added functionality to the start-chain bash script that creates a new price-feeder service alongside any umeed service that is created.
- Each new price-feeder service has a number associated with it just like the umeed services that are created where umeed-1 is the node that umeed-1-pf sends its price transactions to.

### CODE REVIEW

File: scripts/chain/start_chain.sh [hunk 1]
```diff
@@ -235,3 +235,53 @@ do
...
+    RPC=$((16657 + $INC))
+    GRPC=$((9092 + $INC))
+    PF_PORT=$((7171 + $INC))
+
+    # Copy the price-feeder config template and replace variables
+    CONFIG_DIR="${DAEMON_HOME}-${a}/config"
+    PF_CONFIG="${CONFIG_DIR}/price-feeder.toml"
+    cp ../configs/price-feeder.toml $CONFIG_DIR
+
+    UMEE_E2E_PRICE_FEEDER_VALIDATOR=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech val --keyring-backend test --output json | jq .address")
```
Comment (adamewozniak) [inline scripts/chain/start_chain.sh:257]: Let's remove the UMEE_E2E parts of these environment variables:
```suggestion
    PRICE_FEEDER_VALIDATOR=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech val --keyring-backend test --output json | jq .address")
```

File: scripts/chain/start_chain.sh [hunk 2]
```diff
@@ -235,3 +235,53 @@ do
...
+    UMEE_E2E_PRICE_FEEDER_ADDRESS=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech acc --keyring-backend test --output json | jq .address")
```
Comment (adamewozniak) [inline scripts/chain/start_chain.sh:258]:
```suggestion
    PRICE_FEEDER_ADDRESS=$(eval "umeed keys show validator${a} --home ${DAEMON_HOME}-${a} --bech acc --keyring-backend test --output json | jq .address")
```

**Review by adamewozniak — APPROVED** [review]
just one small nit but lgtm! 🥳

File: scripts/chain/start_chain.sh (hunk 1 above)
Comment (zarazan) [reply scripts/chain/start_chain.sh:257]: Sounds good, they are a little long. I'll update them in the price-feeder config as well.

File: scripts/chain/start_chain.sh (hunk 2 above)
Comment (zarazan) [reply scripts/chain/start_chain.sh]: https://github.com/umee-network/qa-integration/pull/9/commits/556403f5d3f9f8eb7b6e5a38d88a8d720473f01b

**Pull request merged**